schemas/DatabaseField.schema

```diff
@@ -2,13 +2,14 @@
   domain: { domain: 'Nomen' },
   category: { domain: 'Nomen' },
   required: { domain: 'Logical' },
-  unique: { domain: 'Logical' },
-  index: { domain: 'Logical' },
-  default: { domain: 'Nomen' },
+  unique: { domain: 'Logical' }, // single field unique key
+  index: { domain: 'Logical' }, // single field index
+  default: { domain: 'Nomen' }, // default value to be casted to domain
```

Comment:
What's the meaning of `casted` here? Maybe `used`?

Reply:
I mean type casting